static/app/views/dashboardsV2/widgetCard/widgetCardContextMenu.tsx

```diff
@@ -153,6 +183,24 @@ function WidgetCardContextMenu({
         placement="bottom right"
         disabledKeys={disabledKeys}
       />
+      {showWidgetViewerButton && (
+        <OpenWidgetViewerButton
+          aria-label={t('Open Widget Viewer')}
+          priority="link"
+          size="zero"
+          icon={<IconExpand size="xs" />}
+          onClick={() => {
+            if (!isWidgetViewerPath(location.pathname)) {
```

Comment:
should we create a reusable handle function or a reusable component?

Reply:
made a reusable function!